src/main/kotlin/no/nav/syfo/api/maxdate/SykepengerMaxDateRest.kt

```diff
@@ -22,9 +22,10 @@ fun Route.registerSykepengerMaxDateRestApi(
 
         try {
             val sykepengerMaxDate = sykepengerMaxDateService.getSykepengerMaxDate(sykmeldtFnr)
-                ?.let { it1 -> formatDateForLetter(it1.forelopig_beregnet_slutt) }
+            val maxDate = sykepengerMaxDate?.let { formatDateForLetter(it.forelopig_beregnet_slutt) }
+            val createdAt = sykepengerMaxDate?.let { formatDateForLetter(it.opprettet.toLocalDate()) }
```

Comment:
Jeg er litt usikker på om man skal bruke `opprettet` (dato for når utbetalingen ble gjort) eller `utbetalt_tom` her, kanskje sjekke med Lisa?

Reply:
@kaja-nav Du har rett. Jeg misforstod litt. Fikset nå 👍

Reply:
Du bør høre med Lisa, for jeg er usikker. i Trello står det "siste utbetalingsdato", og det kan jo bety begge deler.

Reply:
@kaja-nav Har allerede gjort det. Hun svarte:"Det er riktig. Vi skal vise utbetalt tom datoen som er datoen maksdatoen ble beregnet også skal vi vise maksdatoen etterpå."